Description: Python binding built failed
 Python binding is built with the libzinnia in system path (should use the
 libzinnia in current build environment), and if libzinnia has never been
 installed, python binding will built failed.
Author: LI Daobing <lidaobing@debian.org>
Forwarded: https://sourceforge.net/tracker/?func=detail&aid=3019030&group_id=232651&atid=1086998
Last-Update: 2010-06-21

--- zinnia-0.06.orig/python/setup.py
+++ zinnia-0.06/python/setup.py
@@ -7,5 +7,7 @@ setup(name = "zinnia-python",
       py_modules=["zinnia"],
       ext_modules = [Extension("_zinnia",
                                ["zinnia_wrap.cxx",],
+                               include_dirs=[".."],
+                               library_dirs=["../.libs"],
                                libraries=["zinnia"])
                      ])
